pkgs/by-name/ie/ietf-cli/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+  rsync,
+  nix-update-script,
+}:
+python3.pkgs.buildPythonApplication rec {
+  name = "ietf-cli";
+  version = "1.27";
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "paulehoffman";
+    repo = "ietf-cli";
+    tag = version;
+    hash = "sha256-D62E0aHVwB2e+3ZNNMOB2E93Q2WKhbWrYtnH76ZOepM=";
+  };
+  buildInputs = [ rsync ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin/
+    install -p ./ietf $out/bin/ietf
```

I don't think -p does quite what you want...

>        -p, --preserve-timestamps
>              apply access/modification times of SOURCE files to
>              corresponding destination files

```suggestion
    install -Dm755 ./ietf $out/bin
```

---

ahh i see. I thought I needed it for reproducibility but you're right, the timestamp is set to 01.01.1970 in the output regardless of the `-p` flag.
I've applied the suggestion.